core/src/main/java/io/seata/core/store/GlobalTransactionDO.java

```diff
@@ -24,23 +25,23 @@
  *
  * @author zhangsen
  */
-public class GlobalTransactionDO {
+public class GlobalTransactionDO implements GlobalTransactionModel {
 
     private String xid;
 
-    private Long transactionId;
+    private long transactionId;
```

Comment:
why changed to basic data types? the transactionId may be null

Reply:
the transactionId will not be null unless manually modified to null